lineapy/plugins/jinja_templates/dvc_dockerfile.jinja

```diff
@@ -0,0 +1,24 @@
+FROM python:{{ python_version }}
+
+RUN mkdir /tmp/installers
+WORKDIR /tmp/installers
+
+# Copy all the requirements to run current DAG
+COPY ./{{ pipeline_name }}_requirements.txt ./
+
+# Install git and dvc
+RUN apt update
+RUN apt install -y git
+RUN pip install dvc
+
+# Install required libs
+RUN pip install -r ./{{ pipeline_name }}_requirements.txt
+
+WORKDIR /home
+COPY . .
+
+# Initialize workdir as a dvc repo
+RUN git init
```

Comment:
we need to init as git here?

Reply:
Yeah. https://dvc.org/doc/command-reference/init 

> DVC works best in a Git repository. This enables all features, providing the most value

So I made it a git repo.